lib/hooks/queries/useMarketStage.ts

```diff
@@ -24,6 +24,7 @@ export const useMarketStage = (market?: Market<Context>) => {
     },
     {
       enabled: Boolean(sdk && isRpcSdk(sdk) && market),
+      refetchInterval: 12 * 1000,
```

Comment:
Is this every 12 seconds? Seems heavy to be given that market status changes very rarely